arviz/rcparams.py

```diff
@@ -9,11 +9,22 @@
 from collections.abc import MutableMapping
 from pathlib import Path
 from typing import Any, Dict
+from typing_extensions import Literal
+
+NO_GET_ARGS: bool = False
+try:
+    from typing_extensions import get_args
+except ImportError:
+    NO_GET_ARGS = True
```

Comment:
It looks like we can simplify all this if we require py>=3.7, which we should already do by next release as we agreed to follow numpy guidance on this. Note that numpy, scipy, pandas, matplotlib... latest versions all require >=3.7.

As how to proceed, it is probably better to merge this to avoid conflicts and having this hanging but after open an issue to fix this once we drop 3.6 (there are already a couple PRs on that).

references:
https://github.com/python/typing/blob/master/typing_extensions/src_py3/typing_extensions.py#L17
https://github.com/python/typing/blob/master/typing_extensions/src_py3/typing_extensions.py#L157

Reply:
Added #1638 for this.